### ISSUE TITLE
mistake in Gumbel softmax sample function.

### ISSUE THREAD

**Issue by sobalgi** [YongfeiYan/Gumbel_Softmax_VAE#3]
The gumbel_softmax_sample function is logits + gumbel_sample. But it should have been F.log_softmax(logits) + gumbel_sample according to the paper. Is this not a mistake?
